_episodes/46.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: episode
+title: "46: Trino heats up with Ignite"
+date: 2023-02-23
+tags: datbase memory in-memory ignite
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Trino Release"
+  desc: "408-411"
+  time: 
+- title: "Concept of the episode"
+  desc: "Introducing the Ignite connector to Trino"
+  time: 
+- title: "Concept of the episode"
+  desc: "What is Ignite?"
+  time: 
+- title: "Demo of the episode"
+  desc: "Using Ignite with Trino"
+  time: 
+- title: "Concept of the episode"
+  desc: "Contributing the Ignite connector"
+  time: 
+- title: "PR of the episode"
+  desc: "Add support for `migrate` procedure in Iceberg"
+  time: 
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Jason (please comment however you'd like this to introduce you)
```

Comment:
[Jason](https://www.linkedin.com/in/jian-chen-7aa3a2225) Senior data engineer at Shopee.